_sass/minimal-mistakes/_sidebar.scss

```diff
@@ -89,6 +89,24 @@
       position: sticky;
       top: 2em;
       float: right;
+
+      .toc {
+        .toc__menu {
+          overflow-y: auto;
+          scrollbar-width: thin;
+          max-height: calc(100vh - 7em);
+          &::-webkit-scrollbar {
+            width: 8px;
+          }
+          &::-webkit-scrollbar-track {
+            background: #F0F0F0;
```

Comment:
Is this color needed? I want to avoid a situation where this scrollbar doesn't match the native browser one applied to the entire page.